### PR TITLE
Add section in readme for debugging samples

### PR DESCRIPTION
**Pull request by hui1110**
As title.

### CODE REVIEW

File: eventhubs/spring-cloud-azure-starter-integration-eventhubs/eventhubs-integration/README.md [hunk 1]
```diff
@@ -136,15 +136,23 @@ source ./terraform/setup_env.sh
  . terraform\setup_env.ps1
 ```
 
+If you want to run the program in debug mode, you can save the output value. 
+
 ## Run Locally
 
+### Run the program directly
```
Comment (backwind1233) [inline eventhubs/spring-cloud-azure-starter-integration-eventhubs/eventhubs-integration/README.md:143]: I suggest not using `program` here, use sample instead.

File: eventhubs/spring-cloud-azure-starter-integration-eventhubs/eventhubs-integration/README.md [hunk 2]
```diff
@@ -136,15 +136,23 @@ source ./terraform/setup_env.sh
...
+
 In your terminal, run `mvn clean spring-boot:run`.
 
 
 ```shell
 mvn clean spring-boot:run
 ```
 
+### Run the program in debug mode
```
Comment (backwind1233) [inline eventhubs/spring-cloud-azure-starter-integration-eventhubs/eventhubs-integration/README.md:152]: use with title format `Run the Program in Debug Mode`

File: eventhubs/spring-cloud-azure-starter-integration-eventhubs/eventhubs-integration/README.md [hunk 3]
```diff
@@ -136,15 +136,23 @@ source ./terraform/setup_env.sh
  . terraform\setup_env.ps1
 ```
 
+If you want to run the program in debug mode, you can save the output value.
```
Comment (backwind1233) [inline eventhubs/spring-cloud-azure-starter-integration-eventhubs/eventhubs-integration/README.md:139]: better add a picture to demonstrate here.

File: eventhubs/spring-cloud-azure-starter-integration-eventhubs/eventhubs-integration/README.md [hunk 4]
```diff
@@ -136,15 +136,23 @@ source ./terraform/setup_env.sh
...
 In your terminal, run `mvn clean spring-boot:run`.
 
 
 ```shell
 mvn clean spring-boot:run
 ```
 
+### Run the program in debug mode
+
+You can debug your program by adding the saved output values or finding the required parameters in the `terraform.tfstate` file in the program's terraform directory and adding them to the tool's environment variables or the program's `application.yaml` file.
```
Comment (backwind1233) [inline eventhubs/spring-cloud-azure-starter-integration-eventhubs/eventhubs-integration/README.md:154]: suggest delete this
image


and better add a pic to demonstrate how to do it.

File: eventhubs/spring-cloud-azure-starter-integration-eventhubs/eventhubs-integration/README.md (hunk 2 above)
Comment (backwind1233) [reply eventhubs/spring-cloud-azure-starter-integration-eventhubs/eventhubs-integration/README.md:152]: replace "program", it's too general.

**Review by backwind1233 — CHANGES_REQUESTED** [review]

File: eventhubs/spring-cloud-azure-starter-integration-eventhubs/eventhubs-integration/README.md [hunk 5]
```diff
@@ -136,15 +136,25 @@ source ./terraform/setup_env.sh
  . terraform\setup_env.ps1
 ```
 
+If you want to run the program in debug mode, you can save the output value. 
+
+![image](resource/output.png)
+
 ## Run Locally
 
+### Run the sample directly
```
Comment (backwind1233) [inline eventhubs/spring-cloud-azure-starter-integration-eventhubs/eventhubs-integration/README.md:145]: Run the Sample Directly

File: eventhubs/spring-cloud-azure-starter-integration-eventhubs/eventhubs-integration/README.md [hunk 6]
```diff
@@ -136,15 +136,25 @@ source ./terraform/setup_env.sh
  . terraform\setup_env.ps1
 ```
 
+If you want to run the program in debug mode, you can save the output value. 
+
+![image](resource/output.png)
```
Comment (backwind1233) [inline eventhubs/spring-cloud-azure-starter-integration-eventhubs/eventhubs-integration/README.md:141]: it's hard work if we place a png file in each sample, I am wondering whether we can do it just like below
image

**Review by backwind1233 — CHANGES_REQUESTED** [review]

File: eventhubs/spring-cloud-azure-starter-integration-eventhubs/eventhubs-integration/README.md [hunk 7]
```diff
@@ -136,15 +136,29 @@ source ./terraform/setup_env.sh
  . terraform\setup_env.ps1
 ```
 
+If you want to run the program in debug mode, you can save the output value.
```
Comment (backwind1233) [inline eventhubs/spring-cloud-azure-starter-integration-eventhubs/eventhubs-integration/README.md:139]: program --> sample

File: cache/spring-cloud-azure-starter/spring-cloud-azure-sample-cache/README.md [hunk 8]
```diff
@@ -123,14 +123,32 @@ source ./terraform/setup_env.sh
...
+
+```shell
+AZURE_SUBSCRIPTION_ID=...
+AZURE_RESOURCE_GROUP=...
+AZURE_CACHE_REDIS_NAME=...
+```
+
 ## Run Locally
 
+### Run the sample Directly
```
Comment (saragluna) [inline cache/spring-cloud-azure-starter/spring-cloud-azure-sample-cache/README.md:136]: how about change to Run with Maven

File: cache/spring-cloud-azure-starter/spring-cloud-azure-sample-cache/README.md [hunk 9]
```diff
@@ -123,14 +123,32 @@ source ./terraform/setup_env.sh
...
+
 In your terminal, run `mvn clean spring-boot:run`.
 
 ```shell
 mvn clean spring-boot:run
 ```
 
+### Run the sample in Debug Mode
```
Comment (saragluna) [inline cache/spring-cloud-azure-starter/spring-cloud-azure-sample-cache/README.md:144]: how about changing to Run in IDEs

**Review by backwind1233 — APPROVED** [review]
LGTM

**Pull request merged**